package.json

```diff
@@ -9,40 +9,52 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@aws-amplify/form-generator": "^1.0.3",
+    "@aws-amplify/model-generator": "^1.0.9",
     "@splidejs/react-splide": "^0.7.12",
     "@splidejs/splide": "^4.1.4",
     "@splidejs/splide-extension-auto-scroll": "^0.5.3",
-    "axios": "^1.7.7",
+    "axios": "^1.7.8",
     "clsx": "^2.1.1",
-    "framer-motion": "^11.5.4",
+    "core-js": "^3.39.0",
+    "framer-motion": "^11.12.0",
+    "glob": "^11.0.0",
     "gsap": "^3.12.5",
     "install": "^0.13.0",
-    "lenis": "^1.1.13",
+    "lenis": "^1.1.17",
     "lottie-react": "^2.4.0",
-    "lucide": "^0.439.0",
-    "lucide-react": "^0.439.0",
-    "next": "~14.2.10",
-    "npm": "^10.8.3",
+    "lucide": "^0.462.0",
+    "lucide-react": "^0.462.0",
+    "next": "^15.0.3",
+    "npm": "^10.9.1",
     "react": "^18",
```

Comment:
can we upgrade to react 19?

Reply:
That is not officially released yet, it is a release candidate (RC) for the last 7-8 months

Reply:
that makes sense. thank you!